### ISSUE TITLE
histone modification browser datasets

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1441]
Genome-wide mapping of histone modifications and mass spectrometry reveal H4 acetylation bias and H3K36 methylation at gene promoters in fission yeast.

Sinha I, Buchanan L, Rönnerblad M, Bonilla C, Durand-Dubief M, Shevchenko A, Grunstein M, Stewart AF, Ekwall K.

Epigenomics. 2010 Jun;2(3):377-93. doi: 10.2217/epi.10.18.

PMID: 22121899

**Comment by mah11** [comment]
> @mah11 in progress? 

did you look in the browser?

**Comment by ValWood** [comment]
Long read transcriptome (Jeff Boake lab) would complement other transcriptome data (https://pubmed.ncbi.nlm.nih.gov/27856494/)

**Comment by ValWood** [comment]
intron data (mainly meiosis specific)
Widespread exon skipping triggers degradation by nuclear RNA surveillance in fission yeast
https://genome.cshlp.org/content/25/6/884.short

**Comment by ValWood** [comment]
I'm reviewing the action items from 2022 SAB. This is the only outstanding task. Maybe this can be prioritised (the 3 datasets requested after )CT 2022, not all of them). @kimrutherford  could you show @PCarme  how to do this at some point?

**Comment by kimrutherford** [comment]
> I think it will be straightforward to turn that table into a browser track.

The processed data file in BED format is here:
https://www.pombase.org/external_datasets/Bitton_2015_PMID_25883323_introns/PMID_25883323_Supplemental_Tables-S12.sorted.bed.gz

**Comment by kimrutherford** [comment]
I've had a look at the other two datasets.

> Long read transcriptome (Jeff Boake lab) would complement other transcriptome data (https://pubmed.ncbi.nlm.nih.gov/27856494/)

For this paper I think it makes sense to show the assembled transcripts as a track, but I can only find the raw reads (in GEO).  We can show raw reads but it's probably not what users would want.  Also we'd need to align the reads to the genome to make a BAM file.

> Also, https://pubmed.ncbi.nlm.nih.gov/35618415/
> suggested by @juanmatacambridge

This one looks easier.  They have various GTF files on figshare and a BAM file.  I approve!

One of the files is "Spom_novel_transcripts.gtf" which sounds immediately useful:
https://figshare.com/articles/dataset/Native_RNA_sequencing_in_fission_yeast_reveals_frequent_alternative_splicing_isoforms/19368146?file=34398125

We'd need to read the paper to decide which of the other files would make good browser tracks.

**Comment by kimrutherford** [comment]
> Widespread exon skipping triggers degradation by nuclear RNA surveillance in fission yeast
> https://genome.cshlp.org/content/25/6/884.short
> Table S12

That track is now on the site:
https://www.pombase.org/reference/PMID:25883323
https://www.pombase.org/jbrowse/?loc=I%3A2819691..2838230&tracks=Forward%20strand%20features%2CReverse%20strand%20features%2CNovel%20introns%20%E2%80%93%20Bitton%20et%20al.%20(2015)&highlight=

We might need to work on how it looks.  We can configure colours, labels and sizes of features:

![image](https://github.com/user-attachments/assets/e0c76048-e446-4fa8-8377-caf767ac489c)

**Comment by ValWood** [comment]
One small thing.
I think we should have a new data type "intron-related".
It would group this dataset and the branch point data.

The label "genome sequence and features" was intended to be  reserved for the actual genome sequence and annotated features (maybe we could improve that label?


<img width="651" alt="Screenshot 2024-11-27 at 08 36 21" src="https://github.com/user-attachments/assets/eb3bf820-5839-4498-adb3-5ce79a7cec20">

**Comment by PCarme** [comment]
> We might need to work on how it looks. We can configure colours, labels and sizes of features:

We could maybe change the labels to be simpler/more readable. 
Also, the data contains 3 "types" of introns: novel introns, novel introns with non canonical splice sites, and known introns. Would it be possible to display the different types in different colours ?
Do you have something else in mind ?

**Comment by ValWood** [comment]
The different types as different colours would be nice.

A lot of the novel introns look like "known introns" to me!

**Comment by PCarme** [comment]
> A lot of the novel introns look like "known introns" to me!

They seem to be detected as known introns only when their start and end coordinates perfectly match the annotated coordinates. A lot of these look like known introns with slightly displaced start or end.

**Comment by ValWood** [comment]
Ah ok... I noted a few that look as though they could be small missing genes. We can have a look at these one day...

**Comment by PCarme** [comment]
> This one looks easier. They have various GTF files on figshare and a BAM file. I approve!
> 
> One of the files is "Spom_novel_transcripts.gtf" which sounds immediately useful:

Agreed, this one sounds good. They also have a file with novel transcript isoforms not annotated on PomBase (stated in the article), and files mapping a few unannotated 5' and 3' UTRs.

**Comment by kimrutherford** [comment]
> Also, the data contains 3 "types" of introns: novel introns, novel introns with non canonical splice sites, and known introns. Would it be possible to display the different types in different colours ?

I forgot about this.  I'll do it next.

**Comment by kimrutherford** [comment]
Hi @PCarme

The two GFF tracks are ready to go.  The URLs are:
  - https://www.pombase.org/external_datasets/Monta%c3%b1%c3%a9s_2022_PMID_35618415/Spom_isoforms.sorted.gff.gz
  - https://www.pombase.org/external_datasets/Monta%c3%b1%c3%a9s_2022_PMID_35618415/Spom_novel_transcripts.sorted.gff.gz

I've had a look at the UTR files.  I couldn't find any cases where the annotated PomBase UTR doesn't match the UTRs from this dataset.  So maybe it's not so useful as a JBrowse track.

@ValWood what do you think?

**Comment by ValWood** [comment]
>I've had a look at the UTR files. I couldn't find any cases where the annotated PomBase UTR doesn't match the UTRs from this dataset. So maybe it's not so useful as a JBrowse track.

I expected the 5' UTRs to match well, but not to be perfect. That's a bit of a shock!

I think its a useful track as confirmation from an alternative source. Presumably this data is glucose minimal media?

There should still be introns in utrs that were missed.

the 3'UTRS won't match up I'm sure, they are from random sources, and include some long readthroughs

**Comment by ValWood** [comment]
from our FAQ


3’ UTR features use four data sources and a set of precedence criteria:

Highest priority is given to data from low-throughput “conventional” experiments performed on individual mRNAs and reported in publications or submitted to EMBL. Where low-throughput data are not available, one of three high-throughput datasets is used.
The Broad data published in 2011 by Rhind et al. ([PMID:21511999](http://www.ncbi.nlm.nih.gov/pubmed?term=21511999)) is given precedence because it is the most recent, is higher resolution and detected splicing within the UTRs. Note: This study used a “greedy” algorithm to determine the longest possible transcript from transcriptome reads, which may result in the prediction of longer UTRs than are actually present. Use these data with caution, and refer to the transcript profiling data in the genome browser for genes of interest.
For genes not covered by (1) or (2), start/end data from Lantermann et al. ([PMID:20118936](http://www.ncbi.nlm.nih.gov/pubmed?term=20118936)) based on transcriptome data from Dutrow et al. ([PMID:18641648](http://www.ncbi.nlm.nih.gov/pubmed?term=18641648)) are used where available.
For genes not covered by (1), (2) or (3), we use data from Wilhelm et al. ([PMID:18488015](http://www.ncbi.nlm.nih.gov/pubmed/18488015)).

**Comment by kimrutherford** [comment]
> There should still be introns in utrs that were missed.

I had a quick look in the data files.  None of the UTRs have introns.

So you think we should host both tracks from that dataset, 5' UTRs and 3' UTRs?

**Comment by ValWood** [comment]
I'm not sure what is available and what is being hosted?
We should probably include all tracks?
the title is Native RNA sequencing in fission yeast reveals frequent alternative splicing isoforms
so I expected more than canonical UTRs

**Comment by kimrutherford** [comment]
> I'm not sure what is available and what is being hosted?
> We should probably include all tracks?

There are 4 datafiles that we're looking at:

 - **Spom_isoforms.gtf**: Annotation file for genes with alternative isoforms not annotated in Pombase.
 - **Spom_novel_transcripts.gtf**: Annotation file for novel transcripts not annotated in Pombase.
 - **Spom_dRNA_5UTR_Stringtie.gtf**: Genomic coordinates of new 5'UTR regions in S. pombe.
 - **Spom_dRNA_3UTR_Stringtie.gtf**: Genomic coordinates of new 3'UTR regions in S. pombe.

We think the first two sound useful and are not sure about the two UTR datasets.

> the title is Native RNA sequencing in fission yeast reveals frequent alternative splicing isoforms so I expected more than canonical UTRs

Sorry, my description above wasn't clear.

Each of the UTRs in the datasets are different from the PomBase UTRs.  For the 5' the starts are different from PomBase and for the 3' the ends are different.  None of them seem to change the translated AA sequence.

**Comment by ValWood** [comment]
I would include them all. What are the conditions here? that might account for the differences (and of course the ends are often heterogenous to a few bases) , but it will be useful  to host them all for comparisons.

**Comment by PCarme** [comment]
The conditions are "S. pombe (strain CBS5682) was grown in a rich medium at 30°C and harvested during log-phased growth", and here is the composition of their medium found in the supplemental data
<img width="576" alt="image" src="https://github.com/user-attachments/assets/9025e539-4993-4695-a990-fa9b07bde0e4">
So it looks like YPD

**Comment by PCarme** [comment]
> Hi @PCarme
> 
> The two GFF tracks are ready to go. The URLs are:

 I have added the tracks for these two files to the JBrowse configuration file

**Comment by kimrutherford** [comment]
> I have added the tracks for these two files to the JBrowse configuration file

Thanks!

I'll check how things look tomorrow.

**Comment by kimrutherford** [comment]
Thanks Pascal.  The two new tracks are in the browser.

I haven't added the UTR tracks yet.

**Comment by kimrutherford** [comment]
> I haven't added the UTR tracks yet.

Those two tracks are [now on pombase.org](https://www.pombase.org/jbrowse/?loc=I%3A1094491..1109940&tracks=Forward%20strand%20features%2CReverse%20strand%20features%2CNovel%20isoforms%20%E2%80%93%20Monta%C3%B1%C3%A9s%20et%20al.%20(2022)%2CNovel%20transcripts%20%E2%80%93%20Monta%C3%B1%C3%A9s%20et%20al.%20(2022)%2C5%20prime%20UTRs%20%20%E2%80%93%20Monta%C3%B1%C3%A9s%20et%20al.%20(2022)%2C3%20prime%20UTRs%20%20%E2%80%93%20Monta%C3%B1%C3%A9s%20et%20al.%20(2022)&highlight=).

I've added two rows to the `jbrowse_track_metadata.csv` file.  Could you check that it's OK?

**Comment by PCarme** [comment]
It seems all good to me !

**Comment by ValWood** [comment]
This is odd. The 5'UTRs are labelled as 5'UTRs:

<img width="567" alt="Screenshot 2024-12-09 at 09 48 45" src="https://github.com/user-attachments/assets/f50e329f-a855-48b6-b15d-7cdc98fca212">

**Comment by ValWood** [comment]
This is the tracks to host next
The dynamic landscape of fission yeast meiosis alternative-splice isoforms
[Zheng Kuang](https://pubmed.ncbi.nlm.nih.gov/?sort=date&term=Kuang+Z&cauthor_id=27856494) [1](https://pubmed.ncbi.nlm.nih.gov/27856494/#full-view-affiliation-1) [2](https://pubmed.ncbi.nlm.nih.gov/27856494/#full-view-affiliation-2), [Jef D Boeke](https://pubmed.ncbi.nlm.nih.gov/?sort=date&term=Boeke+JD&cauthor_id=27856494) [1](https://pubmed.ncbi.nlm.nih.gov/27856494/#full-view-affiliation-1) [2](https://pubmed.ncbi.nlm.nih.gov/27856494/#full-view-affiliation-2), [Stefan Canzar](https://pubmed.ncbi.nlm.nih.gov/?sort=date&term=Canzar+S&cauthor_id=27856494) [3](https://pubmed.ncbi.nlm.nih.gov/27856494/#full-view-affiliation-3) [4](https://pubmed.ncbi.nlm.nih.gov/27856494/#full-view-affiliation-4)
Affiliations Expand
PMID: 27856494

(It's the one mentioned by Li-Lin )

**Comment by PCarme** [comment]
> This is odd. The 5'UTRs are labelled as 5'UTRs:

Although they look like 5'UTRs 
<img width="802" alt="image" src="https://github.com/user-attachments/assets/4668b32f-901d-4ba9-aca3-f8d465c94b9d">

**Comment by ValWood** [comment]
Agreed, maybe they are incorrectly labelled in the file?

**Comment by ValWood** [comment]
Small thing with metadata:

<img width="831" alt="Screenshot 2024-12-09 at 09 55 53" src="https://github.com/user-attachments/assets/32508fd1-c8af-42cb-8bd2-c45926a7692f">

danny button's "intron-related" hads n/a for WT or mutant , I assume it was wild-type (or  even exosome mutant to get the transcripts that are normally degraded?) 

Was this  "vegetative growth"

**Comment by kimrutherford** [comment]
> Agreed, maybe they are incorrectly labelled in the file?

Yep!  Sorry about that.  I made a typo while processing.  It's fixed now, if you shift-reload.

**Comment by ValWood** [comment]
I just checked the abstract and I see it's pooled. MAybe we should have  a way to depict this?
It's a shame that WT and mutant datasets are mixed.  It would be good to make it clear that RNA processing mutants are included because this shows all of the cryptic transcripts that are normally degraded...
"yet most of these splicing variants were cryptic and increased in nuclear degradation mutants."

It's a shame because this dataset would be *MUCH* more useful (to us and others)  if  we had separate datasets for the processing mutants

**Comment by PCarme** [comment]
I have written "Identification of unnanotated introns in RNA-seq data from 116 samples of various genotypes and growth phases" in the "short_description" column of the metadata file, but I notice now that this column is not displayed on JBrowse. Maybe we could put an indication that the samples are pooled in the "Comment" column ?

**Comment by ValWood** [comment]
I wonder what we do with "short description" then?
We can discuss this tomorrow, but this is minor tweaking; you can go ahead and announce these...

**Comment by kimrutherford** [comment]
> I wonder what we do with "short description" then?

It's just used for internal documentation.  Some of the data for the older tracks came from when PomBase was collaborating with Ensembl Genomes.  I think "short_description" is from that time.

**Comment by ValWood** [comment]
@PCarme  here I would transfer the table to a new ticket and 
provide an updated list of all of the outstanding tasks. Its a bit long to read!
